README.md

```diff
@@ -251,8 +251,8 @@ If you want to use the ONNXInfer as the bandwidth estimator, you should specify
 
     ONNXInfer:
     ``` shell
-    sudo docker run -d --rm -v `pwd`/examples/peerconnection/serverless/corpus:/app -w /app --name alphartc alphartc peerconnection_serverless receiver.json
-    sudo docker exec alphartc peerconnection_serverless sender.json
+    sudo docker run -d --rm -v /home/wang/AlphaRTC/examples/peerconnection/serverless/corpus:/app -w /app --name="alpha" alphartc peerconnection_serverless sender_pyinfer.json
```

Comment:
I think adding an environment variable would be a better option.
How about adding the following line, instead of putting `/home/wang/AlphaRTC/` directly on the README?

```
export ALPHARTC_HOME=/path/to/AlphaRTC
sudo docker run -d --rm -v ${ALPHARTC_HOME}/examples/peerconnection/serverless/corpus:/app -w /app --name alphartc alphartc peerconnection_serverless sender_pyinfer.json
```

Reply:
This is a good choice and I agree with your suggestion.